src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/personopplysninggrunnlag/PersonopplysningGrunnlagServiceTest.kt

```diff
@@ -136,11 +141,12 @@ internal class PersonopplysningGrunnlagServiceTest {
     }
 
     @Test
-    fun `oppdaterPersonopplysningGrunnlag - skal deaktivere eksisterende aktivt personopplysningsgrunnlag og opprette et nytt`() {
+    fun `oppdaterPersonopplysningGrunnlag - skal deaktivere eksisterende aktivt personopplysningsgrunnlag og opprette et nytt med barn fra søknad i FGB`() {
```

Comment:
FGB står vel for førstegangsbehandling? oppdaterPersonopplysningGrunnlag skal vel opprette nytt personopplysningsgrunnlag med barn fra forrige behandling